### ISSUE TITLE
Add portable version of nteract

### ISSUE THREAD

**Issue by ghost** [nteract/archived-desktop-app#1887]
As mentioned in #1873 I would like to run `nteract` in an environment(aka my high-school network) in which I have very few permissions. 

**I would like to run `nteract` in a portable way as I can with [Atom](https://github.com/atom/atom/issues/2939)**. For this I need to store all programs and configurations files for `nteract` and kernels in selected path.

The `zip` version allow this, the problem is that running `nteract` some folders are created, and I'd like to choose the place to put them or eventually with a cli-flag `--portable`, `nteract` would store them in the same folder as the executable. Any thougths.

**Comment by rgbkrk** [comment]
Yeah this would be really nice. I'd love for us to support portable versions, especially for use in schools.

**Comment by rgbkrk** [comment]
What are all the steps we need to take for this @benjaminabel?

**Comment by ghost** [comment]
I don't really know all the settings that need nteract to work, but to make nteract portable it would require to say to the program to search for all settings in a specific folder, for example, visual-studio-code use `data` folder [see doc](https://code.visualstudio.com/docs/editor/portable). 

This is only allowed in zip download, you add a `data` dir in the same folder and put your settings there.

In my linux machine I can find these folders, but maybe nteract uses others:

- `$HOME/.config/nteract`
- `$HOME/.ipython`
